airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -654,10 +637,30 @@ definitions:
               - GET
               - POST
         default: GET
-      request_options_provider:
+      request_body_data:
```

Comment:
Can you add `description` to each of these fields? We're trying to get in the habit of having every field marked this way, so anytime we edit existing fields we should add them if it doesn't already exist. The existing component might have the parameters described in the docstring

Reply:
Done